kitsune/sumo/email_utils.py

```diff
@@ -15,13 +17,48 @@
 log = logging.getLogger("k.email")
 
 
+def normalize_gmail(email: str) -> str:
+    """
+    Return the given email with periods removed from its "local part"
+    if it's a gmail address, otherwise return it unchanged.
+    """
+    if email.lower().endswith("@gmail.com"):
+        # Periods are ignored in the "local part" of gmail addresses.
+        # We need to remove them before using Django's "validate_email",
+        # otherwise it might incorrectly raise a ValidationError.
+        local_part, domain = email.rsplit("@", maxsplit=1)
+        return f"{local_part.replace('.', '')}@{domain}"
+    return email
+
+
+def is_valid_email(email: str) -> bool:
+    """
+    Returns True if the given email address is valid, False otherwise.
+    """
+    try:
+        validate_email(normalize_gmail(email))
+    except ValidationError:
+        return False
+    return True
+
+
 def send_messages(messages):
-    """Sends a bunch of EmailMessages."""
+    """Sends a bunch of email messages."""
     if not messages:
         return
 
+    # Only send each message to its valid recipients,
+    # excluding messages without any valid recipients.
+    cleaned_messages = []
+    for message in messages:
+        # Remove invalid emails and normalize gmails.
+        cleaned_to = [normalize_gmail(email) for email in message.to if is_valid_email(email)]
```

Comment:
You don't need to call the `normalize_email` b/c `is_valid_email` already does that